.travis.yml

```diff
@@ -25,18 +25,16 @@ script:
 
 before_deploy:
   # decrypt ReTest secret key
-  - openssl aes-256-cbc -K $encrypted_58a7d4f10478_key -iv $encrypted_58a7d4f10478_iv
-      -in retest-gmbh-gpg.asc.enc -out retest-gmbh-gpg.asc -d
+  - openssl aes-256-cbc -K "$encrypted_58a7d4f10478_key" -iv "$encrypted_58a7d4f10478_iv"
+    -in retest-gmbh-gpg.asc.enc -out retest-gmbh-gpg.asc -d
   # import decrypted ReTest secret key
   - gpg --fast-import retest-gmbh-gpg.asc
-  # package, sign and deploy artifacts
-  - ${MVN} deploy -DskipTests -P sign
+  # package sign artifacts
+  - ${MVN} verify -DskipTests -Psign
```

Comment:
Looks like we also have to adapt the `pom.xml` since to profile looks different [compared to recheck-web](https://github.com/retest/recheck-web/blob/master/pom.xml#L290).